scripts/install_python38.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+yum install -y openssl-devel bzip2-devel libffi-devel xz-devel wget
```

Comment:
I think it's best to install dependencies outsite of the script - this way Docker layer caching works better. For example, one can try out different python versions / build flags and Docker will use the cached layer which already has the dependencies installed. You can add a `yum install` lint in the Dockerfile, or add a separate `python_env.sh` which installs the deps and is invoked before the build script, like we ahve for the rest.